src/components/molecules/product-card/components/Card.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import styled from '@emotion/styled';
+import { Container } from './Container';
+
+export const Card = ({
+  imgSrc,
+  disable,
+  children,
+  href,
+  onClick,
+  ...props
+}) => {
+  const CardBackground = styled.div`
+    background-image: url(${imgSrc});
+    background-color: inherit;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: block;
+    flex-flow: column;
+    overflow: hidden;
+    position: absolute;
+    transition: 0.3s all;
+    top: -20px;
+    left: 0px;
+    width: 275px;
+    height: 400px;
```

Comment:
could we get some constants for these common values

Reply:
👍

Reply:
Stuck the internal values (ie card height and width) into a constants file. Left global style constants, such as colours or fonts, for another PR.